py-polars/polars/series/series.py

```diff
@@ -1120,7 +1120,7 @@ def __array_ufunc__(
         if method == "__call__":
             if not ufunc.nout == 1:
                 raise NotImplementedError(
-                    "only ufuncs that return one 1D array are supported"
+                    "only `ufunc` variants that return one 1D array are supported"
```

Comment:
[ufunc](https://numpy.org/doc/stable/reference/ufuncs.html) is not a code snippet, shouldn't get backticks.